lib/transform/propertySetup.js

```diff
@@ -37,13 +37,13 @@ function propertySetup(property, name, path) {
   // Only do this once
   if (!property.original) {
     // Initial property setup
-    // Keep the original object properties so we can key off them in the transforms
+    // Keep the original object properties by which we can key off them in the transforms
     to_ret.original = _.clone(property);
-    // Copy the name so we can have a base name to transform
+    // Copy the name by which we can have a base name to transform
     to_ret.name = to_ret.name || name || '';
     // Create an empty attributes object that we can transform on it later
     to_ret.attributes = to_ret.attributes || {};
-    // An array of the path down the object tree so we can use it to build readable names
+    // An array of the path down the object tree by which we can use it to build readable names
```

Comment:
// An array of the path down the object tree; we will use it to build readable names